### ISSUE TITLE
skip_ansible_lint tag ignored on shell command

### ISSUE THREAD

**Issue by ianw** [ansible-community/ansible-lint#1292]
##### Summary
```
$ .tox/linters/bin/ansible-lint --version
ansible-lint 4.3.7
```

test playbook (note hard tab, required)

```yaml
- hosts: "review"
  tasks:

    - name: Tab
      tags:
        - skip_ansible_lint
      shell:
        executable: /bin/bash
        cmd: |
          echo "{{ group_json.id }}	CI-tools" >> groups
```

```
$ .tox/linters/bin/ansible-lint ./test.yaml 
WARNING  Listing 1 violation(s) that are fatal
test.yaml:10: [E203] Most files should not contain tabs
You can skip specific rules or tags by adding them to your configuration file:                                                                       

┌────────────────────────────────────┐
│ # .ansible-lint                                                                      
│ warn_list:  # or 'skip_list' to silence them completely          
│   - '203'  # Most files should not contain tabs                      
└────────────────────────────────────┘
```

I would expect this would be skipped

##### Issue Type

- Bug Report

**Comment by ssbarnea** [comment]
I think that this bug has nothing to do with a particular rule or task type, in fact is likely caused by the fact that `skip_ansible_lint`appears to not longer work in general, including on master branch.

A PR to address this would be very welcome, especially that I can usually only find time to do development on it only during weekends.
